File: ArithmeticCheckPrimeFast.cpp
// Arithmetic.cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include "CheckPrimeFast.h"

#include <iostream>

using namespace std;


int main() {
	try {

		cout << "\n    Welcome to our demonstration sequels\n";
		cout << "Hope you enjoy (and follow) the lessons.\n\n";


		unsigned int start = 1, stop = 100;

		/*
		* Use the Test Prime class.
		*/
		CheckPrimeFast answer(98);
		// Try the fast version
		cout << "\n" << answer.verifyPrimeFast() << "\n";

	}	catch (exception& e) {
		cout << "\n" << e.what() << "\n";
	}
    return 0;
}

File: Arithmetic_CheckPrimeFast.java
package arithmetic;

public class Arithmetic {

    public static void main(String[] args) {
        System.out.println("Welcome to our demonstration sequels");
        System.out.println("Hope you enjoy (and follow) the lessons.");
        System.out.println("");
        
        /*
        * Test for the primeness of a number.
        */
        CheckPrime new_test = new CheckPrime(97); // Is 97 is prime number?
        System.out.println(new_test.verifyPrimeFast());

        System.out.println();
    }

}
